code/API_definitions/CAMARA/number_verification.yaml

```diff
@@ -78,7 +78,7 @@ paths:
         '401':
           $ref: '#/components/responses/Unauthenticated'
         '403':
-          $ref: '#/components/responses/NotAuthenticatedByIpPermissionDenied'
+          $ref: '#/components/responses/PhoneNumberVerificationPermissionDenied'
```

Comment:
The more generic error descriptions are there, the better, so this change is good.